styles/_variables.scss

```diff
@@ -0,0 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@300;500&display=swap');
```

Comment:
This is great that you have these sass variables! We should continue using these for every feature, as well as add some mixins.

Reply:
This looks amazing and matched the wireframe design perfectly! Most of the minor changes were already pointed out, so just compliments from me.